src/model.py

```diff
@@ -147,9 +149,28 @@ def init_engine(self):
         self.setup_lora()
 
         # Create an AsyncLLMEngine from the config from JSON
-        self.llm_engine = AsyncLLMEngine.from_engine_args(
-            AsyncEngineArgs(**self.vllm_engine_config)
-        )
+        aync_engine_args = AsyncEngineArgs(**self.vllm_engine_config)
+        self.llm_engine = AsyncLLMEngine.from_engine_args(aync_engine_args)
+
+        # Create vLLM custom metrics
+        if not aync_engine_args.disable_log_stats:
```

Comment:
We need to document this behavior

Reply:
Agree. I will update the doc in another PR after this gets merged due to time constraint.

Reply:
Do we need the documentation in 24.08? It needs to go in today if needed.

Reply:
I will follow up with another docs PR. Should be cherry-picked with histogram.

Reply:
Docs added